src/main/java/com/libertymutualgroup/herman/aws/ecs/loadbalancing/EcsLoadBalancerHandler.java

```diff
@@ -124,21 +120,36 @@ public LoadBalancer createLoadBalancer(EcsClusterMetadata clusterMetadata, EcsPu
             elbClient.createLoadBalancer(createLoadBalancerRequest);
         } catch (DuplicateLoadBalancerNameException e) {
             LOGGER.debug("Error creating ELB: " + appName, e);
+
+            LoadBalancerDescription loadBalancerDescription = elbClient.describeLoadBalancers(
+                    new DescribeLoadBalancersRequest().withLoadBalancerNames(appName))
+                .getLoadBalancerDescriptions().get(0);
+            if (!elbScheme.equals(loadBalancerDescription.getScheme())) {
+                throw new AwsExecException(String.format("Expected scheme (%s) and actual scheme (%s) do not match. "
+                    + "Load balancer %s must be re-created.", elbScheme, loadBalancerDescription.getScheme(), appName));
```

Comment:
I could find a way to convert from one scheme to another for ELBs, so forcing folks to re-create if scheme is not what's expected. Same was added to the ELBv2 handler.